msg/estimator_innovations.msg

```diff
@@ -0,0 +1,41 @@
+uint64 timestamp		# time since system start (microseconds)
+
+# GPS
+float32[2] gps_hvel	# horizontal GPS velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32    gps_vvel	# vertical GPS velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32[2] gps_hpos	# horizontal GPS position innovation (m) and innovation variance (m**2)
+float32    gps_vpos	# vertical GPS position innovation (m) and innovation variance (m**2)
+
+# External Vision
+float32[2] ev_hvel	# horizontal external vision velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32    ev_vvel	# vertical external vision velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32[2] ev_hpos	# horizontal external vision position innovation (m) and innovation variance (m**2)
+float32    ev_vpos	# vertical external vision position innovation (m) and innovation variance (m**2)
+
+# Fake Position and Velocity
+float32[2] fake_hvel	# fake horizontal velocity innovation (m/s) and innovation variance ((m/s)**2)
+float32    fake_vvel	# fake vertical velocity innovation (m/s) and innovation variance ((m/s)**2)
+float32[2] fake_hpos	# fake horizontal position innovation (m) and innovation variance (m**2)
+float32    fake_vpos	# fake vertical position innovation (m) and innovation variance (m**2)
+
+# Height sensors
+float32 rng_vpos	# range sensor height innovation (m) and innovation variance (m**2)
+float32 baro_vpos	# barometer height innovation (m) and innovation variance (m**2)
+
+# Auxiliary velocity
+float32[2] aux_hvel	# horizontal auxiliar velocity innovation from landing target measurement (m/sec) and innovation variance ((m/sec)**2)
+float32    aux_vvel	# vertical auxiliar velocity innovation from landing target measurement (m/sec) and innovation variance ((m/sec)**2)
+float32[2] flow		# flow innvoation (rad/sec) and innovation variance ((rad/sec)**2)
```

Comment:
Put this under its own comment, instead of auxvel?

Reply:
Going to add this.